crates/sui-framework/tests/delegation_tests.move

```diff
@@ -33,9 +33,7 @@ module sui::delegation_tests {
 
             let ctx = test_scenario::ctx(scenario);
 
-            // Create two delegations to VALIDATOR_ADDR_1.
-            sui_system::request_add_delegation(
-                system_state_mut_ref, coin::mint_for_testing(10, ctx), VALIDATOR_ADDR_1, ctx);
```

Comment:
Test failed here because the pool tokens are distributed in LIFO order (vector::pop_back) of delegation so the staked SUI value and delegation value don't match when we get these two objects using `test_scenario::take_last_created_owned`. Removing one of the delegations here to make the test pass. I will later change test_scenario so we can do more than just taking the last created object and write more holistic tests.